apply-notch-hax.sh

```diff
@@ -19,6 +35,8 @@ if [ ! -e $DIFF ]; then
        then fork the repo, create a patches/$DEVICE.diff
        and modify this script to fetch the patches from your
        fork for testing!"
+	   echo "Alternatively, specify a device template to use with
```

Comment:
Could this echo be merged with the one above it? (the script also needs to be called without the `.diff` suffix)
```bash
		echo "ERROR: It seems your device isn't supported by this project;
       please tune the files on your device manually first,
       then fork the repo, create a patches/$DEVICE.diff
       and modify this script to fetch the patches from your
       fork for testing!
       
       Alternatively, specify a device template to use with
       '$0 <device>'"
```